Controller/ContentTypeController.php

```diff
@@ -217,7 +232,7 @@ public function viewContentTypeAction($contentTypeId, $languageCode = null)
         $deleteForm = $this->createForm(new ContentTypeDeleteType(), ['contentTypeId' => $contentTypeId]);
 
         return $this->render('eZPlatformUIBundle:ContentType:view_content_type.html.twig', [
-            'language_code' => $languageCode,
+            'language_code' => $this->getVerifiedContentTypeLanguageCode($contentType, $languageCode),
```

Comment:
How is this used?

Can't we just simply verify  $languageCode against translations contentType is in and give error/warning to user and fallback to first language? aka

```php
if (!isset($content->names[$languageCode])) {
    // do something else then prev logic as we can't use $languageCode
}
```